App/server/routes/scheduled_actuator.py

```diff
@@ -0,0 +1,97 @@
+import os
+import sys
+from typing import List
+
+parent = os.path.abspath(".")
+sys.path.append(parent)
+
+print(sys.path)
```

Comment:
remove print, also is the path appending still needed here?

Reply:
Doesn't seem necessary